### ISSUE TITLE
Translations

### ISSUE THREAD

**Issue by MarkL1961** [CorsixTH/CorsixTH#335]
I don't know if you have noticed, but the translators used to get updates in a google group https://groups.google.com/forum/#!forum/corsix-th-translators
When there were new or changed strings, it was reported there.

This became broken once the link was removed to update corsixth code in google code.

Does anyone know how to either mend or change this to work from github?

It could explain why the translations are behind.

**Comment by bobcanthelpyou** [comment]
@SimplyRyan could you update the profile of CorsixTH at Transifex for the v0.50 release? Also there are some new requests for new languages. Are there some other people that could take care of it?

**Comment by SimplyRyan** [comment]
On my honeymoon at the moment, will take a look at this when I get back. Creating a workflow and adding administrators to transifex is on my todo list.

**Comment by Alberth289346** [comment]
@SimplyRyan  any news? If this or corsix as a whole dropped off your list, that's fine, but please let us know

**Comment by SimplyRyan** [comment]
Hello, I just started to get back into the swing of things actually, had a busy year (moved house, got married etc.). Will get moving on things in the coming weeks. Had a few people contact me via the forums, helped where possible and pointed them in the right direction so that's encouraging on the translation front.

And a final note, @bobcanthelpyou its important to note we can't host the full core languages online due to copyright issues (then obviously Transifex would be a doddle! ).

**Comment by lewri** [comment]
@SimplyRyan it's been a few years since any update but I noticed that transifex is no longer free (if it ever was).

Do you have any suggestions on how to move forward with our translations? There was a fresh revisit to this topic in #1657 now the forums are gone.

**Comment by SimplyRyan** [comment]
We have a grandfathered in Transifex account (they still support free/open source projects but it seems you no longer sign up directly for these types of accounts). Happy to invite anyone so they can look around. The site is here: https://www.transifex.com/corsixth/corsixth/

Its a useful tool and I'm sure we could make some use out of it, for example, all the strings for the installer are online and translation can be submitted and tracked. 

Regarding the future I guess the same problems still remain now as they did in 2016:

- Core language strings need to be kept out of these systems due to copyright meaning maintaining a source language and seeing gaps is tricky. Hosting our added strings is possible for sure, but its only a small percentage of the translation
- Complex 'difference/override' in the core languages makes the barrier for entry tricky for new translators
- File format not particularly friendly, makes automated workflows hard to implement and error-prone

I will try and take a look at the wiki page and update it, I helped a few people by email so I'm sure I've got some content and screenshots I can dump.
